docs/Benchmarks.md

```diff
@@ -0,0 +1,94 @@
+# Benchmarks
+
+This document explains how to benchmark ASP.NET Core changes. It is intended for Microsoft employees.

```

Comment:
Should this say that the document explains how to benchmark changes _in the aspnetcore perf lab_?